lib/chat/chat.dart

```diff
@@ -65,7 +65,7 @@ buildChatView() {
       buildHeader(),
```

Comment:
It's not letting me highlight the lines but in lines 36 and 47, you can probably place those string constants in a constant file, if youd like